spark/src/test/java/org/apache/zeppelin/spark/SparkInterpreterTest.java

```diff
@@ -111,15 +111,17 @@ public void testSparkSql(){
 		repl.interpret("val people = sc.parallelize(Seq(Person(\"moon\", 33), Person(\"jobs\", 51), Person(\"gates\", 51), Person(\"park\", 34)))\n", context);
 		assertEquals(Code.SUCCESS, repl.interpret("people.take(3)", context).code());
 
-		// create new interpreter
-		Properties p = new Properties();
-		SparkInterpreter repl2 = new SparkInterpreter(p);
-		repl2.open();
-
-		repl.interpret("case class Man(name:String, age:Int)", context);
-		repl.interpret("val man = sc.parallelize(Seq(Man(\"moon\", 33), Man(\"jobs\", 51), Man(\"gates\", 51), Man(\"park\", 34)))", context);
-		assertEquals(Code.SUCCESS, repl.interpret("man.take(3)", context).code());
-		repl2.getSparkContext().stop();
+		if (repl.getSparkContext().version().startsWith("1.1")) { // spark 1.2 or later does not allow create multiple SparkContext in the same jvm by default.
```

Comment:
Same comment as below, potentially safer to explicitly check the numerical value of the version, e.g.:
```
String[] split = version.split(".")
return Integer.parseInt(split[1]) > 3;
```